src/d3a/d3a_core/sim_results/device_statistics.py

```diff
@@ -49,8 +50,19 @@ def _device_price_stats(cls, area_dict: Dict, subdict: Dict, core_stats, current
         area_core_trades = core_stats[area_dict['uuid']].get('trades', [])
         trade_price_list = []
         for t in area_core_trades:
-            if t['seller'] == area_dict['name'] or t['buyer'] == area_dict['name']:
-                trade_price_list.append(t['energy_rate'] / 100.0)
+            if t['seller'] == area_dict['name']:
```

Comment:
But i also have a general comment here. Why is this only coming up now? 
I guess the one sided market wasn't looked at for a long time. 
I don't think that changing the results here (or also in https://github.com/gridsingularity/d3a/pull/914) really fixes the issue that we count the grid fees only for the bids.
Also, you missed one bid AFAIS; have a look in the video in the story. The difference between offer and trade rate in the offer/bid/trade graph is making me worried. 
I hate to say it but we might need another offline discussion about this.